Lmod/branches/7.0/lmod-bash-xtrace.patch

```diff
@@ -0,0 +1,13 @@
+--- Lmod-6.6/init/bash.in	2016-12-15 14:01:46.258628788 +0100
++++ Lmod-6.6-new/init/bash.in	2016-12-15 14:04:36.979331682 +0100
+@@ -1,4 +1,10 @@
+ # -*- shell-script -*-
++xtrace_saved="$(set -o | grep xtrace | grep 'on')";
++set +x
++
++[ -n "$xtrace_saved" ] && set -x 
++unset xtrace_saved
++
+ if [[ ${-/x} != $- ]]; then
+    echo "Start of Lmod init/bash script to define the module command"
+ fi
```

Comment:
I'm not sure, but I'd rather not print out anything at all OR print it right after the set +x section.
But, if you do the latter, you'd rather do it conditionally upon `xtrace_saved`, perhaps... no?